### DISCUSSION TITLE
Governance - The framework for supporting Filecoin Plus

### DISCUSSION THREAD

**Discussion by Kevin-FF-USA** [filecoin-project/Allocator-Governance#256] [Ideas]
**Submitting this discussion for community feedback and input in an effort to clarify and standardized some of the key governance topics associated with Fil+. Seeking feedback to bring up to Governance Team (Galen) for review and revision in 2025.**

--- 
The Filecoin Plus program is intended to onboard high quality data to a distributed network, which operates under a governance framework to ensure trust and collaboration. Governance principles for the program outline 

**Defined Roles and Rights:** 
- Responsibilities for applicants, clients, and Allocators in ensuring standards are met and maintained.
- **Accountability and Transparency:** Regular audits and transparent reporting of datacap refresh and applications for new pathways. Operationally, governance should balance stability and innovation. 

**Key practices include:**
- Formal Proposal Processes: Community discussions - either in Slack, Github, or bi-weekly Fil+ Governance meeting, for issues and proposed changes.
- **Consensus:** Major changes, such as security patches or adjustments, should require broad agreement and thorough testing.

By focusing on standards and accountability, **_Fil+ should evolve while maintaining its core mandate - to onboard quality data to the Filecoin network._**

## 1. New Application (for Allocators) Submission Guidelines
- **Standards for Submission**:
  - Visit the [Allocator Governance GitHub page](https://github.com/filecoin-project/Allocator-Governance).
  - Create a new pull request by adding a new file to the repository.
  - Name it appropriately,  "Allocator application <org name>".
   - Commit the change and add it as a branch, then click "Create pull request".
- **Required Documentation**:
  - A comment will appear with a link to the Airtable, which serves as your actual application form.
   - Open this link in a new tab for easy access.
   - See the [blog from FIDL](https://blog.allocator.tech/2024/05/rolling-applications-are-open-for.html) on their experience/input on the application process

## 2. New Application (for Allocators) Review and Approval 
- **Community Review  System**:
  - All new applications are received and reviews on their ability to add value to the Filecoin Network.
  - Pathways are prioritized on their ability to onboard data as well as providing a novel approach ([outside of manual)](https://blog.allocator.tech/2024/04/allocator-tech-blog.html).
- **Approval Tiers**:
  - RFA (pathway other than manual) with an existing readme and established MVP - immediate.
  - Pathways which focus on a human verification - as needed.
  - Applications for new Pathways should be for a pathway that is ready to operate. Pathways which need to develop tooling in order to operate are encouraged to obtain a small amount of testing DataCap (DC). 

## 3. Use of Governance Tools
These tools do NOT need to be used, but are recommended as verified and helpful ways to ensure compliance.  If a pathway wishes to utilize their own tooling or a 3rd party tooling, they should meet the spirit of standards set forth.
- **KYC**:
  - Applications which claim to function as a MARKET or AUTOMATED should not rely on a human verification of clients or organizations.  While no specific tooling is mandated to participate in Fil+ as an Allocator, the tooling for verifying your clients should be - clearly defined - able to produce a record - [capture the  KYC include the customer identification program (CIP), customer due diligence (CDD), and enhanced due diligence (EDD)](https://www.investopedia.com/terms/k/knowyourclient.asp#:~:text=Know%20Your%20Client%20(KYC)%20is%20a%20standard%20used%20in%20the,enhanced%20due%20diligence%20(EDD).)
- **CID Checker**:
  - Automated checks for testing as part of the submission process. These will be used as one of the primary sources of truth for verification of standards setup in the application to be maintained.
- **SPARK**:
  - Automated checks for testing as part of the submission process. These will be used as one of the primary sources of truth for verification of standards setup in the application to be maintained.

## 4. Stakeholder Involvement
- **Crypto-Specific Oversight**:
  - If working with clients in paid storage deals, deals should have a minimum value that is viewable on-chain.
  - Contacts with clients can be verified as a one off, but should not be utilized as a primary manual verification of deals.
  - Allocators and Clients should operate in good faith. In event of breach of standards in applications, review and removal from the Filecoin Plus program will be conducted by the Filecoin Foundation Governance Team.  In the event a pathway is found to have NOT operated in good, it will be denied access to additional DataCap for disbursement. 
  
## 5. Existing Allocator DatCap Refresh
After DataCap is awarded to a pathway, in order to [receive a refresh the pathway must submit a request.](https://github.com/filecoin-project/Allocator-Governance/issues/new?assignees=Kevin-FF-USA&labels=&projects=&template=-datacap-refresh--.md&title=%5BDataCap+Refresh%5D+%3C1st%2F2nd%2F3rd%3E+Review+of+%3CAllocator+Organization+Name%3E) This request will be reviewed by the Community (Watchdog may serve to compile and comment in place of community if now comments are received), Governance, and Root Key Holders.

- **Security Audits**:  
  SPARK is the primary verification of retrievability.  If a pathway wishes to demonstrate their own metrics, examples, or programs for verifying retrievability - they must meet the same standards as SPARK. 

- **Compliance Standards**:  
  Adhere to requirements that were established in the application to become an Allocator. Failure to adhere to standards (Bookkeeping, Retrievability, SP accounting) may result in - Guidance from Governance - Throttling of DataCap refresh - Removal as Allocator.  Reasons for compliance will be noted in the request for additional DataCap.   

## 6. Community Accountability
- **Key Expectations for Allocators:**  

  **a. High Retrieval Expectations**  
  - Allocators must prioritize high retrieval rates for data (75% as verified in SPARK).  
  - Ensuring data is easily accessible is a critical responsibility.  
  - Allocators are responsible to ensuring that clients operate within the standards the Allocator outlined in their application.

  **b. No Excessive Padding of Datasets**  
  - Metadata and supporting information are allowed, but padding exceeding 25% of a dataset is not acceptable.  
  - Datasets should be appropriately sized, focused, and free of unnecessary filler content.  

  **c. Standardized Bookkeeping**  
  - Allocators must maintain clear bookkeeping, ideally through GitHub for transparency and community access.  
  - If not using GitHub, follow another clear and organized bookkeeping process.  Process should be open and made available to public as well as Governance for refresh audit. 

  **d. Clear Communication of Intent**  
  - Clearly state your intentions and objectives in the application process.  
  - Stay aligned with your declared goals and maintain transparent, well-documented communication.  
  - Maintain onboarding of client data to remain active in the program (as an Allocator)
  - Bi-Weekly Governance meetings will serve as the community source of truth for discussions and forum.
      - [Fil+ Shared Calendar](https://calendar.google.com/calendar/embed?src=c_k1gkfoom17g0j8c6bam6uf43j0%40group.calendar.google.com&ctz=America%2FLos_Angeles%5B%5D)
      - Allocator Governance Meetings are recorded and posted to [Youtube](https://www.youtube.com/playlist?list=PLp3zrT1ewY0kYN1hJpERMUxTCbFC4yZwN)

  **e. Follow a Standardized Format**  
  - Use a consistent and standardized format for all steps in the allocation process, including applications, data submission, retrieval, and compliance reporting.  

  **f. Fee Structures**  
  - Fees are allowed but must be communicated transparently.  
  - Allocators must clearly state any associated costs and the rules governing them.  
  - Paid deals should include an on-chain record.

  **g. Clients and Multiple Allocators**  
  - Applying to multiple allocators is acceptable, but transparency is required.  
  - Allocators should verify a client is not onboarding duplicate data to multiple Allocator Pathways
  - Avoid conflicts of interest and provide clarity to all parties involved.  

  **h. Act in the Interest of Filecoin**  
  - Always evaluate actions by asking: *"Is this growing the Filecoin network and contributing to its success?"*  
  - If the answer is an honest yes, you are likely operating within the spirit of the Filecoin community and guidelines.

**Comment by gitgavinzhao** [comment]
FIL+ should give self an end time same as D.O.G.E.
